File: 268-missing-number/solution.py
# Runtime: 104ms, Beats 93.83% of users with Python3
# Memory: 17.81MB, Beats 64.73% of users with Python3

class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n=len(nums)
        return n*(n+1)//2 - sum(nums)

File: 268-missing-number/solution.cpp
// Runtime: 7ms, Beats 98.02% of users with c++
// Memory: 20.38MB, Beats 16.53% of users with c++

class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int given_sum=0;
        int n=nums.size();
        for(int i=0;i<n;i++)
        {
            given_sum+=nums[i];
        }
        return n*(n+1)/2 - given_sum;
    }
};